### PR TITLE
Added precision support in cmd client

### PR DESCRIPTION
**Pull request by sbouchex**
Added precision support in cmd client in order to have the ability to export timestamp in any support format (secondes, hour...)

### CODE REVIEW

**Comment by otoolep** [comment]
Thanks @sbouchex -- have you signed the CLA?

**Comment by beckettsean** [comment]
fixes https://github.com/influxdb/influxdb/issues/2675

**Comment by sbouchex** [comment]
@otoolep, yep, already signed.

**Comment by otoolep** [comment]
+1 from me. @corylanou ?

**Comment by otoolep** [comment]
Confirmed CLA signed.

File: cmd/influx/main.go [hunk 1]
```diff
@@ -30,6 +30,8 @@ var (
 const (
 	// defaultFormat is the default format of the results when issuing queries
 	defaultFormat = "column"
+	
+	defaultPrecision = ""
```
Comment (otoolep) [inline cmd/influx/main.go]: Actually, the default precision is actually nanos, right? So can you set this correctly?

**Comment by otoolep** [comment]
1 question what the default should actually be.

File: cmd/influx/main.go (hunk 1 above)
Comment (corylanou) [inline cmd/influx/main.go]: +1 for defaulting to nano

**Comment by sbouchex** [comment]
OK, fine, I can change to nano and I will add an option to return 'RFC3339' since it can be understood more easily.

So to confirm:
- 'ns' is the default precision in the command line client only
- 'rfc3339' option is added to support RFC3339 time format

Client library part is not changed

**Comment by otoolep** [comment]
Works for me. @corylanou ?

**Comment by corylanou** [comment]
+1

**Pull request merged**